desc/basis.py

```diff
@@ -238,7 +243,7 @@ def __init__(self, L, sym="even"):
         self._M = 0
         self._N = 0
         self._NFP = 1
-        self._sym = sym
+        self._sym = bool(sym) if not sym else str(sym)
```

Comment:
why make this a string?

Reply:
because `sym` is either False or a string specifying the type of symmetry, like "cos" or "sin"